code/modules/clothing/gloves/color.dm

```diff
@@ -195,3 +195,10 @@
 	name = "evening gloves"
 	desc = "A pair of gloves that reach past the elbow."
 	icon_state = "evening_gloves"
+
+/obj/item/clothing/gloves/fingerless

```

Comment:
Those might need to be clipped, so you can't cut their fingers again, and may change their species restrictions?

Reply:
Xenos being unable to wear those is a feature or not?